kibana/macos/opendistro-macos-kibana-build.sh

```diff
@@ -0,0 +1,126 @@
+#!/bin/bash
+
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+# Description:
+# This script generates packages for opendistro-for-elasticsearch-kibana.
+
+# Node-JS binaries under kibana/node which is being used while installing / removing plugin
+
+
+set -e
+
+# Initialize directories
+REPO_ROOT=`git rev-parse --show-toplevel`
+ROOT=`dirname $(realpath $0)`; echo $ROOT; cd $ROOT
```

Comment:
Remove realpath from here too

Reply:
I think this line should be fine, as this is build and we control if the utils can be installed. I removed it from the install script, which is needed to not force the user to install the utils. The other reason for leaving the realpath is to have this script as close as possible to linux script for easier changes in future.